### PR TITLE
RA-1309 - Clinician facing dashboard does not use parameters passed through extension fragmentConfig

### PR DESCRIPTION
**Pull request by ssmusoke**

### CODE REVIEW

**Comment by dkayiwa** [comment]
What do you think of doing some nicer formatting/indention of your changes? 😊

**Comment by ssmusoke** [comment]
@dkayiwa Tried to make it pretty ...

File: omod/src/main/webapp/pages/clinicianfacing/patient.gsp [hunk 1]
```diff
@@ -5,52 +5,63 @@
 %>
 <script type="text/javascript">
     var breadcrumbs = [
-        { icon: "icon-home", link: '/' + OPENMRS_CONTEXT_PATH + '/index.htm' },
-        { label: "${ ui.escapeJs(ui.encodeHtmlContent(ui.format(patient.patient))) }" ,
-            link: '${ ui.urlBind("/" + contextPath + dashboardUrl, [ patientId: patient.patient.id ] ) }'}
+        {icon: "icon-home", link: '/' + OPENMRS_CONTEXT_PATH + '/index.htm'},
```
Comment (dkayiwa) [inline omod/src/main/webapp/pages/clinicianfacing/patient.gsp]: What exactly changed here?

**Comment by dkayiwa** [comment]
Are all these actual changes or simply formatting? Could we do the formatting in another pull request. That way it will be easier for me to tell what exactly changed. ☺️

**Comment by ssmusoke** [comment]
@dkayiwa These are formatting changes - I used the OpenMRS style for IDEA to make the changes. Looks like the initial file was not formatted according to that convention

File: omod/src/main/webapp/pages/clinicianfacing/patient.gsp (hunk 1 above)
Comment (ssmusoke) [inline omod/src/main/webapp/pages/clinicianfacing/patient.gsp]: Formatting using the OpenMRS style formatter - that is why the initial commit only had changes that I made to the page and nothing else

**Comment by dkayiwa** [comment]
Can you just do the changes without the formatting? That was i will easily tell what actually changed besides formatting.

File: omod/src/main/webapp/pages/clinicianfacing/patient.gsp [hunk 2]
```diff
@@ -86,21 +84,15 @@ ${ui.includeFragment("coreapps", "patientHeader", [patient: patient.patient, act
                 </div>
             </div>
             */ %>
-
+            
             <% if (secondColumnFragments) {
-                secondColumnFragments.each {
-                    def configs = [:];
-                    if (it.extensionParams.fragmentConfig != null) {
```
Comment (dkayiwa) [inline omod/src/main/webapp/pages/clinicianfacing/patient.gsp]: Did you intentionally discard all these lines?

**Pull request merged**